community/submit_guide/pr-and-code-quality.md

```diff
@@ -0,0 +1,420 @@
+---
+id: 'pr-and-code-quality.md'
```

Comment:
How about renaming file name to `code-style-and-quality-guide.md` ? 
Please let me know what's your opinion.

Reply:
good idea!